test/pubtator/pubtator.js

```diff
@@ -12,14 +16,70 @@ import { pubtator_7 } from './10.15252_embj.2023113616.json';
-    //*..*
-    pubtator_7  // pmid: 37317646 | 2 organism hints in title { 7227 aka (Drosophila melanogaster): 1 , 6239 aka (Caeonorhabditis elegans): 1 } | 3 organism hints in abstract: { 9606 aka (Homo sapiens): 1 , 7227 aka (Drosophila melanogaster): 1 , 6239 aka (Caeonorhabditis elegans): 1 } | text: [ [ "Drosophila" , "C. elegans"] , [ "human" , "Drosophila" , "C. elegans" ] ]                                
-  ];
\ No newline at end of file
+    //1..* (multiple organism hints in title and abstract)
+    pubtator_7  // pmid: 37317646 | 1 organism hints in title { 7227 aka (Drosophila melanogaster): 1 } | 2 organism hints in abstract: { 9606 aka (Homo sapiens): 1 , 7227 aka (Drosophila melanogaster): 1 } | text: [ [ "Drosophila"] , [ "human" , "Drosophila"] ]                                
+  ];
+
+  describe('pubtator', function(){
+    describe('map', function(){
+      bioCDocuments.forEach( bioCDocument => {
+        const { infons: { doi } } = bioCDocument;
+  
+        describe(`BioC Document ${doi}`, function(){
+          let hints;
+  
+          before( () => {
+            hints = map( bioCDocument );
+          });
+  
+          it('Should map to a non-zero list', function(){
+            expect( hints ).to.have.length.above(0);
+          });
+  
+          it('Should map to a list of Hints', function(){
+            hints.forEach( h => {
+              expect( h ).to.be.an.instanceof( Hint );
+            });
+          });
+  
+          it('Should have valid section', function(){
+            const isValidSection = h => _.includes( SECTION, h.section );
+            expect( hints.every( isValidSection ) ).to.be.true;
+          });
+  
+          it('Should have valid texts field', function(){
+            const hasTexts = h => h.texts instanceof Array;
+            expect( hints.every( hasTexts ) ).to.be.true;
+          });
+  
+          it('Should have unique xref in a given section', function(){
+            const bySectionXref = ({ xref, section }) => `${xref.dbPrefix}_${xref.id}_${section}`;
+            const inTitle = _.filter( hints, ['section', 'title'] );
+            const uniqInTitle = _.uniqBy( inTitle, bySectionXref );
+            const inAbstract = _.filter( hints, ['section', 'abstract'] );
+            const uniqInAbstract = _.uniqBy( inAbstract, bySectionXref );
+            expect( inTitle.length ).to.equal( uniqInTitle.length );
+            expect( inAbstract.length ).to.equal( uniqInAbstract.length );
+          });
+  
+          it(`Should be all an ORGANISM hint`, function(){
+            const isOrganism = h => h.type === HINT_TYPE.ORGANISM;
+            expect( hints.every( isOrganism ) ).to.be.true;
+          });
+            
+          it('Should have xref to NCBI Taxonomy', function(){
```

Comment:
I don't think this is testing if it has a "NCBI Taxonomy". In fact, I'm not sure when this test would ever fail since you're filtering items before you test them.